ui/lobby/src/setupCtrl.ts

```diff
@@ -149,9 +148,45 @@ export default class SetupController {
       aiLevel: this.aiLevel(),
     });
 
+  private savePropsToStoreExceptRating = () => {
+    if (this.gameType && this.store[this.gameType]) {
```

Comment:
but then it won't save anything if `this.store[this.gameType]` doesn't yet exist? Which is not how `savePropsToStore` works. Is that intended?

Reply:
Changed it to follow existing saveProps